### PR TITLE
Initial preparations for start translating the GUI

### PR DESCRIPTION
**Pull request by Senyoret1**
Related to https://github.com/skycoin/skycoin/issues/2242#issuecomment-475961556 https://github.com/skycoin/skycoin/issues/2243 and https://github.com/skycoin/skycoin/issues/1981 (but does not solve them).

Changes:
- This PR adds a script lets know not only which language files should be updated, but also which specific elements need attention.
- For it to work, an specific process for creating and updating the translations must be follow. The details are in the README.md file.

This should make the process of updating the translation files much easier than simply checking them manually.

The current Russian translation will have to be checked and updated manually before being able to check it with the script.

Does this change need to mentioned in CHANGELOG.md?
No

### CODE REVIEW

**Comment by Senyoret1** [comment]
The last commit includes the Chinese translation files. Checking the translation itself is outside the scope of this PR, but the files could help to test the script.

File: src/gui/static/src/assets/i18n/README.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+translation file was updated, some texts of the applications have been changed.
+This means that the translation file should be updated.
+
+At the end of the script excecution, the console will display the list of all
+errors found, if any.
+
+# Update a translation
+
+Before updating a translation file, you should follow the steps of the
+[Checking if a languaje file needs to be updated](#Checking-if-a-languaje-file-needs-to-be-updated)
```
Comment (gz-c) [inline src/gui/static/src/assets/i18n/README.md]: languaje -> language

File: src/gui/static/src/assets/i18n/cn.json [hunk 2]
```diff
@@ -0,0 +1,429 @@
+{
```
Comment (gz-c) [inline src/gui/static/src/assets/i18n/cn.json]: The language code for chinese is zh, not cn. https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

The file should be zh.json

**Comment by Senyoret1** [comment]
Changes made. I added a link to the translations readme in the main readme and in the GUI development readme.

**Pull request merged**